app/src/main/java/com/mealsmadeeasy/ui/home/discover/SearchActivity.kt

```diff
@@ -24,10 +25,11 @@ class SearchActivity : BaseActivity() {
 
     @Inject lateinit var mealStore: MealStore
     private var querySubject : BehaviorSubject<String> = BehaviorSubject.create()
+    private val KEY_SAVED_QUERY = "key saved query"
```

Comment:
Can we use a different value for this string? I typically would use `"SearchActivity.SavedQuery"`